pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -533,6 +533,8 @@ func GetOpenshiftBootstrapClusterRoles() []rbac.ClusterRole {
 			Rules: []rbac.PolicyRule{
 				// pull images
 				rbac.NewRule("get").Groups(imageGroup, legacyImageGroup).Resources("imagestreams/layers").RuleOrDie(),
+				// read signatures
```

Comment:
you could just add "imagestreamimages" to the resources in the rule above this one.

Reply:
wanted to make it clear why that rule is there